packages/familie-backend/src/config.ts

```diff
@@ -1,12 +1,17 @@
 import { error } from './logging';
 import { IAppConfig } from './typer';
 
-export const envVar = (navn: string, påkrevd = true): string => {
-    if (!process.env[navn] && påkrevd) {
+export const envVar = (navn: string, påkrevd = true, defaultValue?: string): string => {
```

Comment:
if påkrevd=== false, defaultValue === undefined and envVriable does not exist, the function returns undefined. Is this correct?

Reply:
No, it should maybe exit I think?

Reply:
No, actually it can return undefined, because it is not required.